<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyViewTestClass</class>
 <widget class="QWidget" name="MyViewTestClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>912</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyViewTest</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0">
    <widget class="QPushButton" name="pushButton_Open">
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="lineEdit">
     <property name="inputMask">
      <string/>
     </property>
     <property name="placeholderText">
      <string>输入绘图对象名称</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QPushButton" name="pushButton_Search">
     <property name="text">
      <string>Serach</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="pushButton_Draw">
     <property name="text">
      <string>Rect</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="7">
    <widget class="QLabel" name="label_Msg">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>14</height>
      </size>
     </property>
     <property name="text">
      <string>此处显示光标坐标与RGB值</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" rowspan="3" colspan="6">
    <widget class="ROIView" name="Viewer" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QPushButton" name="pushButton_Clear">
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="lineEdit_Search">
     <property name="placeholderText">
      <string>输入目标图元对象序列</string>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="QPushButton" name="pushButton_AffineRect">
     <property name="text">
      <string>AffineRect</string>
     </property>
    </widget>
   </item>
   <item row="6" column="4">
    <widget class="QCheckBox" name="checkBox_ByName">
     <property name="text">
      <string>ByName</string>
     </property>
    </widget>
   </item>
   <item row="4" column="5">
    <widget class="QPushButton" name="pushButton_Description">
     <property name="text">
      <string>Description</string>
     </property>
    </widget>
   </item>
   <item row="6" column="5">
    <widget class="QPushButton" name="pushButton_Measure">
     <property name="text">
      <string>Measure</string>
     </property>
    </widget>
   </item>
   <item row="1" column="7" rowspan="3" colspan="6">
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="4" column="10" colspan="3">
    <widget class="QPushButton" name="pushButton_UpdateMSG">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>UpdateMessage</string>
     </property>
    </widget>
   </item>
   <item row="6" column="10" colspan="3">
    <widget class="QComboBox" name="comboBox_item">
     <item>
      <property name="text">
       <string>线段</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>矩形</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>椭圆</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>多边形</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>文字</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>位图</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="7" colspan="3">
    <widget class="QPushButton" name="pushButton_ClearMSG">
     <property name="text">
      <string>ClearMessage</string>
     </property>
    </widget>
   </item>
   <item row="6" column="7" colspan="3">
    <widget class="QPushButton" name="pushButton_Insert">
     <property name="text">
      <string>Insert</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ROIView</class>
   <extends>QWidget</extends>
   <header>roiview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="MyViewTest.qrc"/>
 </resources>
 <connections/>
</ui>
